src/components/organisms/AdvancedLendingOrderCard/AdvancedLendingOrderCard.tsx

```diff
@@ -472,7 +473,19 @@ export function AdvancedLendingOrderCard({
                         />
                         <OrderDisplayBox
                             field='Future Value'
-                            value='--' // todo after apy -> apr
+                            value={
```

Comment:
Have you seen this comment by Aki-san? https://securedfinance.slack.com/archives/C032N8KUNRZ/p1715564449805779?thread_ts=1715523621.537459&cid=C032N8KUNRZ Regarding using `getOrderEstimation`

Reply:
Currently, we do not use orderEstimation when the user is filling in the details of the order. It is used on the order confirmation screen. If the implementation changes later on, we can fix it.